test-suite/src/test/require-jdk8/org/scalajs/testsuite/javalib/time/LocalTimeTest.scala

```diff
@@ -13,6 +13,41 @@ object LocalTimeTest extends TemporalAccessorTest {
   describe("java.time.LocalTime") {
     testTemporalAccessorApi(MAX, MIN, NOON)
 
+    it("should respond to `isSupported`") {
+      for {
+        t <- Seq(MAX, MIN, NOON)
+        f <- ChronoField.values() if f.isTimeBased
+      } {
+        expect(t.isSupported(f)).toBeTruthy
+      }
+
+      for {
+        t <- Seq(MAX, MIN, NOON)
+        f <- ChronoField.values() if !f.isTimeBased
+      } {
+        expect(t.isSupported(f)).toBeFalsy
+      }
+
+      for {
+        t <- Seq(MAX, MIN, NOON)
+        u <- ChronoUnit.values() if u.isTimeBased
+      } {
+        expect(t.isSupported(u)).toBeTruthy
+      }
+
+      for {
+        t <- Seq(MAX, MIN, NOON)
+        u <- ChronoUnit.values() if !u.isTimeBased
+      } {
+        expect(t.isSupported(u)).toBeFalsy
+      }
+
+      for (t <- Seq(MAX, MIN, NOON)) {
+        expect(t.isSupported(null: TemporalUnit)).toBeFalsy
+        expect(t.isSupported(null: TemporalField)).toBeFalsy
+      }
+    }
+
```

Comment:
You should also add some tests on `isTimeBased` and `isDateBased`. There are few cases, you could test them all.